File: src/main/java/com/yah/Hello.java
package com.yah;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello word");
      // new Person().hello();
        Person p = new Person("Bob",72f, 1.69f);
        p.hello();
        p.weight = 72f;
        p.height = 1.69f;
        System.out.println(p.bmi());
        int score = 88;
        System.out.println(score < 80 || score > 90);
        char c = 'A';                     // A = 65
        System.out.println(c > 'a');      // a = 97

        /*int age = 19;
        Integer age2 = 19;
        char c = '我';
        Character C2 = 'A';
        byte b = 120;
        float height = 1.7f;
        boolean adult = true;
        boolean enroll = false;
        String name = "Tom";*/
    }
}

File: src/main/kotlin/com/yah/Hello.kt
package com.yah

import javafx.scene.text.FontWeight

fun main(args: Array<String>) {
//    println("Hello kotlin")
    val h = Human(weight = 72f, height = 1.69f )
    println(h.bmi())
    val score = 88
    println(score > 60)
    val c :Char = 'A'
    println(c.toInt()>60)
  /*  var age = 19
    age = 20
    var weight = 66.5f
    var name : String
    name = "Hank"
*/
}

class Human(var name: String = "",var weight: Float, var height:Float) {
    init {
        println("test $weight" )
    }
    //constructor(name:String,weight: Float,height: Float):this(weight，height)
    fun  bmi():Float{
        val bmi = weight / (height*height)
        return bmi
    }
    fun hello(){
        println("Hello kotlin")
    }

}